test/browser/home-page.test.jsx

```diff
@@ -4,9 +4,107 @@ var React = require('react/addons');
 var TestUtils = React.addons.TestUtils;
 
 var stubContext = require('./stub-context.jsx');
+var StubRouter = require('./stub-router');
 var HomePage = require('../../pages/home.jsx');
 
 var stubBlogFeedLoader = require('./stub-blog-feed-loader');
+var Util = require('../util.js');
+
+describe("HomePage", function() {
+ var homePage, pledgeBtn;
+
+ beforeEach(function() {
+  homePage =  stubContext.render(HomePage);
```

Comment:
just a nit, I'm noticing that there is only 1 space of indentation on these test functions--would be great to have them be 2 spaces so it's consistent w/ the rest of our code!